docs/gambit/gambit.md

```diff
@@ -199,6 +201,22 @@ For example, one of the mutant files for
 uint256 res = decimals ** a;
 ```
 
+Also included in the `out/` directory is a JSON summary of all mutants produced, `out/results.json`.
+The results include the filename and a unique string ID of each mutant, along with
+a brief description and the `diff` between the mutant and the original file. For example,
+
+```json
+[
+  {
+    "description": "<brief summary of mutant>",
+    "diff": "<stdout of `diff` command on the mutant and original file>"
+    "id": "0",
+    "name": "out/path/to/mutant.sol"
+  },
+  ...
+]
+```
```

Comment:
This doesn't parse as json, which is causing the syntax highlighter to fail.  For other examples I just changed to `"..."`

Reply:
Looks like there's a missing comma as well.  Make sure that the readthedocs CI step passes (the spell / link check step is currently known to be broken).